upgrading.md

```diff
@@ -0,0 +1,12 @@
+# Upgrade Procedures
+
+This document describes how registries should add support for new APIs and features.
+
+## Referrers API
+
+The Referrers API here is described by [Listing Referrers](spec.md#listing-referrers) and [end-12a](spec.md#endpoints).
+When registries add support for the Referrers API, this API needs to account for manifests that were pushed before the API was available using the [Referrers Tag Schema](spec.md#referrers-tag-schema).
+
+1. Registries MUST include preexisting Image and Artifact manifests that are listed in an Index tagged with the tag schema and have a valid Refers field.
+1. Registries MAY include all existing Image and Artifact manifests with a Refers field in the API response.
+1. Registries MUST include all new Image and Artifact manifests with a valid Refers field in the API response.
```

Comment:
new is not very specific.. seems to imply some prior version that is not a MUST here..

Reply:
If a manifest is pushed before the API is enabled with a `refers` field, but that manifest was not included in an Index found using the tag schema, then we don't require the registry to scan those manifests for the API response. It dramatically reduces the lift required by a large registry to enable the API since scanning every manifest on their server is expected to be a time and storage IO intensive process.